docker/package/scripts/tezos-endorser-start

```diff
@@ -46,3 +46,7 @@ if [[ -z "$BAKER_ADDRESS_ALIAS" ]]; then
 else
     launch_endorser "$BAKER_ADDRESS_ALIAS" "$@"
 fi
+
+# Waiting till node start to respond
+
```

Comment:
Let's leave a `TODO` that we should use `--keep-alive` instead of this loop once https://gitlab.com/tezos/tezos/-/issues/2873 is fixed